packages/gafl-webapp-service/src/pages/contact/digital-licence/licence-fulfilment/licence-fulfilment.njk

```diff
@@ -1,48 +1,40 @@
 {% extends "standard-form.njk" %}
 {% from "radios/macro.njk" import govukRadios %}
 
-{% set title = mssgs.licence_fufilment_title_you if data.isLicenceForYou else mssgs.licence_fufilment_title_other %}
+{% set title = mssgs.licence_fulfilment_title_you if data.isLicenceForYou else mssgs.licence_fulfilment_title_other %}
 
-{% set errorMsg = mssgs.licence_fufilment_error %}
+{% set errorMsg = mssgs.licence_fulfilment_error %}
 
 {% set errorMap = {
         'licence-option': {
-            'any.required': { ref: '#licence-option', text: errorMsg }
+            'any.required': { ref: '#licence-option', text: mssgs.licence_fulfilment_error }
         }
     }
 %}
 
 {% block pageContent %}
-    <p class="govuk-body">{{ mssgs.licence_fufilment_content_info }}</p>
+    <p class="govuk-body">{{ mssgs.licence_fulfilment_content_info }}</p>
     {{ govukRadios({
         idPrefix: "licence-option",
         name: "licence-option",
         items: [
             {
                 value: "digital",
-                text: mssgs.licence_fufilment_radio_paperless,
+                text: mssgs.licence_fulfilment_radio_paperless,
                 checked: payload['licence-option'] === 'digital',
                 hint: {
-                    html: '<ul class="govuk-list govuk-list--bullet"><li>' + mssgs.licence_fufilment_radio_paperless_hint1 + '</li><li>' + mssgs.licence_fufilment_radio_paperless_hint2 + '</li></ul>'
+                    html: mssgs.licence_fulfilment_radio_paperless_hint_you if data.isLicenceForYou else mssgs.licence_fulfilment_radio_paperless_hint_other
```

Comment:
Please could you do the decision logic in the getData method rather than here? The less untested code we have in templates the better!

Reply:
@jaucourt Had a crack at an alternate implementation in the new commit – let me know what you think.

Reply:
Draft ticket to implement this elsewhere: https://eaflood.atlassian.net/browse/IWTF-3404